running-your-first-ot-server/setup-webserver-linux.md

```diff
@@ -0,0 +1,201 @@
+exit;
+```
+Insert these lines one by one.
+Now we have setup our database superuser :)
+
+## Configure phpmyadmin
+Setup phpmyadmin to access your database via the browser.
+
+Insert the following configuration in a new file inside the nginx folder. The following command will create the file and when you are in insert the configuration posted below.
+( same as for the webserver nginx configuration I have made comments for the ubuntu versions inside the php location )
+`sudo vim /etc/nginx/sites-available/phpmyadmin`
+```bash
+server {
+    listen      2344;
+    server_name  _;
+    root /usr/share/phpmyadmin;
+
+    add_header X-Frame-Options DENY;
+    add_header X-Content-Type-Options nosniff;
+    add_header X-XSS-Protection "1; mode=block";
+
+    client_max_body_size 256M;
+    error_page 404 @notfound;
+
+    location / {
+        index index.html index.php;
+        try_files $uri $uri/ /index.php?$args;
+    }
+
+    location ~* \.(gif|jpg|jpeg|png|bmp|js|css)$ {
+        expires max;
+    }
+
+    location ~ \.php$ {
+        include snippets/fastcgi-php.conf;
+
+    ################## UBUNTU 18.04 LTS ##################
+           fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
+    ################## UBUNTU 20.04 LTS ##################
+    #       fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
+
+    }
+
+    location @notfound {
+        return 404 "You're not browsing correctly.";
+        add_header Content-Type text/plain always;
+    }
+
+}
+```
+Now we have to save the file and quit the file, we do this the following way:
+`CTRL+C` After that you will just write `:wq` press enter and we have saved and quit the file :)
+
+Now we have to enable the phpmyadmin configuration for nginx we do that by running the next command:
+`sudo ln -s /etc/nginx/sites-available/phpmyadmin /etc/nginx/sites-enabled/phpmyadmin`
+
+Finally restart the nginx server with `sudo systemctl restart nginx` and we will be able to access phpmyadmin by http://youripordomain:2344 login with the database username and password we've created.
+
+# Setup your Server for TFS
+First of all we need to install a few packages to our system which makes us able to build the TFS sources.
+
+Directly copy'd from the TFS github wiki(https://github.com/otland/forgottenserver/wiki/Compiling-on-Ubuntu):
+## Install the required software
+
+The following command will install Git, CMake, a compiler and the libraries used by The Forgotten Server.
+
+Git will be used to download the source code, and CMake will be used to generate the build files.
+```bash
+sudo apt-get install git cmake build-essential liblua5.2-dev libgmp3-dev libmysqlclient-dev libboost-system-dev libboost-iostreams-dev libboost-filesystem-dev libpugixml-dev libcrypto++-dev
+```
+## Download the source code
+
+```bash
+git clone --recursive https://github.com/otland/forgottenserver.git
+```
+## Generate the build files
+```bash
+cd forgottenserver
+mkdir build && cd build
+cmake ..
+```
+## Build
```

Comment:
Try to leave one empty line above every header.

Reply:
Done 👍